lib/get_parser.py

```diff
@@ -324,7 +324,23 @@ def get_parser() -> argparse.ArgumentParser:
         action="store_true",
         help="Use jobsubdevgpvm01 etc. to submit",
     )
-
+    parser.add_argument(
+        "--onsite",
+        dest="usage_model",
+        action="store_const",
+        const="OPPORTUNISTIC,DEDICATED",
+        default="OPPORTUNISTIC,DEDICATED,OFFSITE",
+        help="run jobs locally only; " "usage_model=OPPORTUNISTIC,DEDICATED",
+    )
+    parser.add_argument(
+        "--offsite",
+        dest="usage_model",
+        action="store_const",
+        const="OPPORTUNISTIC,DEDICATED,OFFSITE",
```

Comment:
Shouldn't this just be "OFFSITE" if we say `--offsite`?

Reply:
Sorry, my misunderstanding of the request in Issue 49. I will change this.

Reply:
done